account_banking_mandate_contact/tests/test_account_payment_order.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl
+
+from unittest.mock import patch
+
+from odoo import fields
+from odoo.tests.common import Form, TransactionCase
+
+from odoo.addons.account.models.account_payment_method import AccountPaymentMethod
+
+
+class TestAccountPaymentOrder(TransactionCase):
+    def setUp(self):
+        super(TestAccountPaymentOrder, self).setUp()
```

Comment:
PEP 3135 (https://peps.python.org/pep-3135/)

```suggestion
        super().setUp()
```